optuna/integration/pytorch_lightning.py

```diff
@@ -56,10 +56,10 @@ def __init__(self, trial: optuna.trial.Trial, monitor: str) -> None:
         self.is_ddp_backend = False
 
     def on_init_start(self, trainer: Trainer) -> None:
-        self.is_ddp_backend = trainer.accelerator_connector.distributed_backend is not None
+        self.is_ddp_backend = trainer._accelerator_connector.distributed_backend is not None
```

Comment:
[Question] Does pytorch-lightning provide any public interface to check the distributed backend?

Reply:
I'm afraid that I was not able to find such an interface on the [trainer page](https://pytorch-lightning.readthedocs.io/en/latest/api/pytorch_lightning.trainer.trainer.html?highlight=trainer#trainer).

Reply:
That makes sense. Thank you for your investigation.